.travis.yml

```diff
@@ -0,0 +1,3 @@
+language: ruby
+rvm:
+  - 2.2.5
```

Comment:
Travis still needs some config. Likely a dummy `.env` file that matches what it's supposed to do to setup the database for testing.